tuf/api/metadata.py

```diff
@@ -471,15 +520,7 @@ class Snapshot(Signed):
         meta: A dictionary that contains information about targets metadata::
 
             {
-                'targets.json': {
-                    'version': <TARGETS METADATA VERSION NUMBER>,
-                    'length': <TARGETS METADATA FILE SIZE>, // optional
-                    'hashes': {
-                        '<HASH ALGO 1>': '<TARGETS METADATA FILE HASH 1>',
-                        '<HASH ALGO 2>': '<TARGETS METADATA FILE HASH 2>',
-                        ...
-                    } // optional
-                },
+                'targets.json': <MetaFile INSTANCE>
```

Comment:
Place also use `<MetaFile INSTANCE>` as values for `'<DELEGATED TARGETS ROLE 1>'` and `'<DELEGATED TARGETS ROLE 2>'`